src/coreclr/vm/i386/stublinkerx86.cpp

```diff
@@ -3547,49 +3546,6 @@ VOID StubLinkerCPU::EmitDisable(CodeLabel *pForwardRef, BOOL fCallIn, X86Reg Thr
     // jnz RarePath
     X86EmitCondJump(pForwardRef, X86CondCode::kJNZ);
 
-#if defined(FEATURE_COMINTEROP) && !defined(FEATURE_CORESYSTEM)
```

Comment:
`FEATURE_CORESYSTEM` is not defined, so this was not getting compiled in.

Reply:
Okay, but doesn't that mean it was being compiled in, the `defined` is negated?

Reply:
I meant to say `FEATURE_CORESYSTEM` is always defined (https://github.com/dotnet/runtime/blob/main/src/coreclr/clrdefinitions.cmake#L95). It means `!FEATURE_CORESYSTEM` is always false and this is never getting compiled in.